components/common/ButtonLink.vue

```diff
@@ -50,7 +50,7 @@ const getColor = (active: boolean, exact: boolean) => {
     <RuiButton
       v-bind="{
         variant: 'text',
-        type: 'button',
```

Comment:
don't remove `type: 'button'`, otherwise when you click enter on a form, this button will be clicked